docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,131 @@
+# Quickstart
+
+Get started building a Toolpad app!
+
+<p class="description">
+This page will guide you into creating your very first Toolpad application. We will create a "hello world" type of application that will ask for a name and print a greeting.
+</p>
+
+## Prerequisites
+
+Make sure you have a running Toolpad instance.
+
+1. Download the docker compose file
+
+   ```sh
+   curl -LO https://raw.githubusercontent.com/mui/mui-toolpad/master/docker/compose/docker-compose.yml
+   ```
+
+1. Start the docker compose services
+
+   ```sh
+   docker-compose -f docker/compose/docker-compose.yml up -d
+   ```
+
+Toolpad will be accessible under `http://localhost:3000/`.
+
+## Building your first application
+
+1. Open Toolpad by navigating to [http://localhost:3000/](http://localhost:3000/)
+
+   ![Apps overview](/static/toolpad/apps-overview.png)
+
+1. Click "CREATE NEW" and name your application. Confirm by clicking "CREATE"
+
+   ![Toolpad overview](/static/toolpad/step-1.png)
+
+1. Hover over **Component library** and drag **DataGrid** and **Image** components into **Canvas**
+
+   ![Drag components](/static/toolpad/step-2.png)
+
+   Congratulations, we are done building the UI! Now all we need to do is connect it with a data source.
+
+1. Click anywhere inside **Canvas** (except on the components that you just added) to deselect added components
```

Comment:
This is a little UX problem that is probably worth taking a look at. To get to page level options is not very discoverable. I feel like having to deselect something before you can get to the UI to edit something else isn't optimal.
@prakhargupta1 @gerdadesign I believe you already started an effort on this with the bottom panel that you started to design?